src/CommunityToolkit.Maui/Behaviors/PlatformBehaviors/IconTintColor/IconTintColorBehavior.ios.cs

```diff
@@ -6,27 +6,54 @@ namespace CommunityToolkit.Maui.Behaviors;
 public partial class IconTintColorBehavior
 {
 	/// <inheritdoc/>
-	protected override void OnAttachedTo(View bindable, UIView platformView) =>
+	protected override void OnAttachedTo(View bindable, UIView platformView)
+	{
 		ApplyTintColor(platformView, bindable, TintColor);
 
+		bindable.PropertyChanged += OnElementPropertyChanged;
+		this.PropertyChanged += (s, e) =>
```

Comment:
Should we unsubscribe from this event?

Reply:
No need the life time is the same as the behavior, so all will be collected